robocorp-code/vscode-client/src/extension.ts

```diff
@@ -723,6 +726,19 @@ export async function doActivate(context: ExtensionContext, C: CommandRegistry)
     // i.e.: allow other extensions to also use our feedback api.
     C.registerWithoutStub(ROBOCORP_FEEDBACK_INTERNAL, (name: string, value: string) => feedback(name, value));
 
+    C.register(ROBOCORP_PACKAGE_ENVIRONMENT_REBUILD, () => {
+        const call = async () => {
+            const selected = await listAndAskRobotSelection(
+                "Please select the Task/Action Package for which you'd like to rebuild the environment",
+                "Unable to continue because no Action Package was found in the workspace.",
+                { showActionPackages: true, showTaskPackages: false }
+            );
+
+            await resolveInterpreter(selected.filePath);
+        };
+        call();
```

Comment:
Seems a bit strange to create an internal function here and call it exactly once (tip: you can make the outer one `async` by using adding `async` to it -- i.e.: `async () => `).